libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -3730,6 +3730,71 @@ export type QueryCacheEntryResult<TResult> = {
 // @internal
 export type QueryCacheReducer<TQuery extends IDashboardQuery, TResult, TPayload> = CaseReducer<EntityState<QueryCacheEntry<TQuery, TResult>>, PayloadAction<TPayload>>;
 
+// @public
+export interface QueryCatalogAttributes extends IDashboardQuery {
+    // (undocumented)
+    readonly payload: {
+        readonly attributeRefs: ObjRef[];
+    };
+    // (undocumented)
+    readonly type: "GDC.DASH/QUERY.CATALOG.ATTRIBUTES";
+}
+
+// @public
+export function queryCatalogAttributes(attributeRefs: ObjRef[], correlationId?: string): QueryCatalogAttributes;
+
+// @public
+export interface QueryCatalogDateAttributes extends IDashboardQuery {
+    // (undocumented)
+    readonly payload: {
+        refs: ObjRef[];
+    };
+    // (undocumented)
+    readonly type: "GDC.DASH/QUERY.CATALOG.DATE.ATTRIBUTES";
+}
+
+// @public
+export function queryCatalogDateAttributes(refs: ObjRef[], correlationId?: string): QueryCatalogDateAttributes;
+
+// @public
+export interface QueryCatalogDateDatasets extends IDashboardQuery {
+    // (undocumented)
+    readonly payload: {
+        refs: ObjRef[];
+    };
+    // (undocumented)
+    readonly type: "GDC.DASH/QUERY.CATALOG.DATE.DATASETS";
+}
+
+// @public
+export function queryCatalogDateDatasets(refs: ObjRef[], correlationId?: string): QueryCatalogDateDatasets;
+
+// @public
+export interface QueryCatalogFacts extends IDashboardQuery {
+    // (undocumented)
+    readonly payload: {
+        factRefs: ObjRef[];
```

Comment:
inconsistency in naming with other catalog queries (`refs` only vs `factRefs`/`measureRefs`)